help/androidpublisher.md

```diff
@@ -42,67 +44,67 @@ https://developers.google.com/android-publisher/getting_started
                 KeystorePassword = "my password"
                 KeystoreAlias = "my key alias"
             })
-        |> fun file -> file.CopyTo(Path.Combine(androidProdDir, file.Name)) |> ignore
+        |> Seq.iter(fun file -> file.CopyTo(Path.Combine(androidProdDir, file.Name)) |> ignore)
```

Comment:
Yep I assume if you can keep it backwards compatible then it would be really helpful.